src/encoded/static/components/matrix_deeply_profiled.js

```diff
@@ -0,0 +1,576 @@
+                                    {subCategorySum}
+                                </a>
+                            :
+                                <div style={{ backgroundColor: rowCategoryColor }} />
+                        ),
+                    }))),
+                    css: 'matrix__row-category',
+                },
+            ],
+            subCategoryRows,
+            [{
+                rowContent: [
+                    {
+                        content: (
+                            expandableRowCategory ?
+                                <RowCategoryExpander
+                                    categoryId={categoryNameQuery}
+                                    categoryName={rowCategoryBucket.key}
+                                    expanded={categoryExpanded}
+                                    expanderClickHandler={expanderClickHandler}
+                                    expanderColor={rowCategoryTextColor}
+                                    expanderBgColor={rowCategoryColor}
+                                />
+                            : null
+                        ),
+                    },
+                    {
+                        content: null,
+                        colSpan: 0,
+                    },
+                ],
+                css: `matrix__row-spacer${expandableRowCategory ? ' matrix__row-spacer--expander' : ''}`,
+            }]
+        );
+    }, [{ rowContent: header, css: 'matrix__col-category-header' }]);
+    return { dataTable: matrixDataTable, rowKeys: matrixRowKeys };
+};
+
+
+/**
+ * Render the area above the facets and matrix content.
+ */
+const MatrixHeader = ({ context }) => {
+    const visualizeDisabledTitle = context.total > MATRIX_VISUALIZE_LIMIT ? `Filter to ${MATRIX_VISUALIZE_LIMIT} to visualize` : '';
+    const pageUrl = context['@id'];
+
+    return (
+        <div className="matrix-header">
+            <div className="matrix-header__title">
+                <div className="matrix-title-badge">
+                    <h1>{context.title}</h1>
+                </div>
+                <div className="matrix-description">
+                    <div className="matrix-description__text">{matrixDescription}</div>
+                </div>
+            </div>
+            <div className="matrix-header__controls">
+                <div className="matrix-header__filter-controls">
+                    <div className="test-project-selector">
+                        <input type="radio" id="allDeeplyProfiled" name="data-selection" value="All" checked={isAllDeeplyMatrix(pageUrl)} onChange={() => switchDeeplyProfilePageType(true, pageUrl)} />
+                        <label htmlFor="allDeeplyProfiled">All</label> &nbsp; &nbsp;
```

Comment:
I would have used flexbox on `test-project-selector`, and a couple divs (or encapsulate the `input` within the `label`) to generate the margin instead of &nbsp;.